src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -20,17 +20,22 @@
           {{getIdentifier}}</strong> back on the register.
         </p>
 
+        <p v-if="isFilingTypeAdminFreeze">
+          You are about to {{ !isAdminFreeze ? 'Freeze' : 'Unfreeze' }} <strong><span class="text-uppercase">{{getEntityName}}</span>,
```

Comment:
You might as well use [the Vuetify helper class](https://vuetifyjs.com/en/styles/text-and-typography/) to make the text bold.